.github/workflows/downstream.yaml

```diff
@@ -156,4 +156,4 @@ jobs:
     - run: java -version
     - run: sudo apt-get update -y
     - run: sudo apt-get install libxml2-utils
-    - run: .kokoro/downstream-client-library-check.sh ${{matrix.repo}}
+    - run: .kokoro/downstream-client-library-check.sh ${{matrix.repo}} test
```

Comment:
Why are you modifying the non-GraalVM downstream check?

Reply:
The downstream checks will accept a JOB_TYPE as a second argument to distinguish between a [standard run ](https://github.com/googleapis/java-shared-dependencies/blob/410e624ee01da8db69de3df4da45999f1693824a/.kokoro/presubmit/common.cfg#L24) and graalvm run.